examples/timeline/groups/stackSubgroup.html

```diff
@@ -0,0 +1,185 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+    <title>Timeline | Subgroups</title>
+
+    <style>
+        body, html {
+            font-family: arial, sans-serif;
+            font-size: 11pt;
+        }
+        .vis-item.expected {
+            background-color: transparent;
+            border-width: 1px;
+            border-style: dashed!important;
+            border-color: grey;
+            z-index: 0;
+        }
+        .vis-item.vis-selected {
+            opacity: 0.6;
+        }
+        .vis-item.vis-background.marker {
+            border-left: 2px solid green;
+        }
+    </style>
+    <script src="../../../dist/vis.js"></script>
+    <link href="../../../dist/vis.css" rel="stylesheet" type="text/css" />
+</head>
+<body>
+
+<p>This example shows the workings of the subgroups. Subgroups do not use stacking, and only work when stacking is disabled.
```

Comment:
As far as I can see it this example shows two new concepts:
* Make subGroups stackable
* show overlay of elements like descripted in #497

I would recommend integrating the "stackSubgroups" option in the existing "subgroups" example and creating a new example that only shows how to realize the #497 feature. I think this are two separated thinks.